Cineaste/Custom UI/SearchController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SearchController.swift
+//  Cineaste
+//
+//  Created by Felizia Bernutz on 14.07.18.
+//  Copyright © 2018 notimeforthat.org. All rights reserved.
+//
+
+import UIKit
+
+public class SearchController: UISearchController {
+    public override init(searchResultsController: UIViewController?) {
+        super.init(searchResultsController: searchResultsController)
+
+        setup()
+    }
+
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
```

Comment:
can be removed if not needed

Reply:
Same here as above

Reply:
The app will throw this error when we remove this, so this is also needed. 
`Fatal error: Use of unimplemented initializer 'init(nibName:bundle:)' for class 'SearchController'`